### ISSUE TITLE
Bot doesn't seem to be interacting at all

### ISSUE THREAD

**Issue by stevenmcastano** [RocketChat/hubot-rocketchat#149]
I've got a totally new fresh install. I'm able to get the bot to start, login to the rockchat server, join the general channel and seems to be sitting there waiting for messages. Even with respond_to_dm on... nothing happen.

DMs don't work, mentioning the bot name doesn't work... am I missing something??

Here's what I see at the console:

```
steve@beniebot:~/benie.bot$ bin/hubot -a rocketchat
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] DEBUG Loading adapter rocketchat
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO Starting Rocketchat adapter version 1.0.1...
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO Once connected to rooms I will respond to the name: benie.bot
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO Connecting To: rocketchat.*******.com
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO Successfully connected!
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO general
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO Logging In
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO Successfully Logged In
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO Looking up Room ID for: general
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO Joining Room: GENERAL
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO rid:  [ 'GENERAL' ]
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO All rooms joined.
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO Successfully joined room: GENERAL
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO Preparing Meteor Subscriptions..
[Thu Jul 28 2016 01:02:15 GMT-0400 (EDT)] INFO Subscribing to Room: __my_messages__
```

**Comment by bothy4g** [comment]
:+1: Same thing here.

**Comment by NioTeX** [comment]
:+1: We are experiencing the same issues.

**Comment by mgaebler** [comment]
👍  Same here. Docker and local install.

**Comment by mgaebler** [comment]
Tried it at last with version 0.1.4. Seems to work properly.
Unfortunately this is not NPMs latest version.

**Comment by stevenmcastano** [comment]
@mgaebler I used the "yo hubot" method of install, how would I have v0.1.4 that way?

**Comment by mgaebler** [comment]
@stevenmcastano after generating hubot scripts, edit the `packages.json` and replace the `"hubot-rocketchat": "^1.0.1",` line with `"hubot-rocketchat": "git://github.com/RocketChat/hubot-rocketchat.git#v0.1.4",`. After that, run `npm install`.

**Comment by timkinnane** [comment]
Duplicate of #142

**Comment by geekgonecrazy** [comment]
Guys i've released v1.0.3 of the adapter.  Please give it a shot and report back any bugs as new issues. 

Going to go ahead and close this issue.
